virtualizarr/tests/test_writers/test_zarr.py

```diff
@@ -42,7 +42,7 @@ def test_zarr_v3_metadata_conformance(tmpdir, vds_with_manifest_arrays: Dataset)
     assert isinstance(metadata["fill_value"], (bool, int, float, str, list))
     assert (
         isinstance(metadata["codecs"], list)
-        and len(metadata["codecs"]) > 1
+        and len(metadata["codecs"]) == 1
```

Comment:
@abarciauskas-bgse did the expectation change so it is correct that there is one codec now?

Reply:
Yes, this is because of how `serializer` was broken out of `_v3_codec_pipeline` so these are 2 separate objects now. Today I will investigate if this is how it should be, but my guess is that this will become a part of the refactor to use the zarr-python array class instead of Virtualizarr's zarr.py

Reply:
yeah I took a look at #175 and I agree that this is all going away. So it is probably not worth doing too much work around it.